### PR TITLE
fix: ssri version update

### PR DESCRIPTION
**Pull request by gudjong**
# Updates SSRI version

## Why

Fixes security vulnerability

### CODE REVIEW

**Review by sindrig — APPROVED** [review]
LGTM

File: yarn.lock [hunk 1]
```diff
@@ -24841,22 +24841,7 @@ sshpk@^1.7.0:
...
-ssri@^7.0.0:
-  version "7.1.0"
-  resolved "https://registry.yarnpkg.com/ssri/-/ssri-7.1.0.tgz#92c241bf6de82365b5c7fb4bd76e975522e1294d"
-  integrity sha512-77/WrDZUWocK0mvA5NTRQyveUf+wsrIc6vyrxpS8tVvYBcX215QbafrJR3KtkpskIzoFLqqNuuYQvxaMjXJ/0g==
-  dependencies:
-    figgy-pudding "^3.5.1"
-    minipass "^3.1.1"
-
-ssri@^8.0.0, ssri@^8.0.1:
+ssri@8.0.1, ssri@^6.0.1, ssri@^7.0.0, ssri@^8.0.0, ssri@^8.0.1:
```
Comment (maranomynet) [inline yarn.lock:24844]: I worry about the consequences of forcing packages using ssri v6 and v7 to go up one or two major versions.

Things could go bang.

Is this safe?

Comment (maranomynet) [reply yarn.lock:24844]: or was this change in yarn.lock the actual result of `yarn install`??

Comment (gudjong) [reply yarn.lock:24844]: It was the result of yarn install

**Pull request merged**